Userland/Libraries/LibWeb/Painting/SVGTextPaintable.cpp

```diff
@@ -31,6 +31,9 @@ void SVGTextPaintable::paint(PaintContext& context, PaintPhase phase) const
     if (!is_visible())
         return;
 
+    if (!layout_node().computed_values().fill().has_value())
+        return;
```

Comment:
What if `stroke()` has a value though?

Reply:
it will be fine because we don't support stroke for svg text yet.

Reply:
Yeah all right :)